src/lazy-loader.marko

```diff
@@ -7,18 +7,24 @@ class {
       componentRenderer: null,
       renderMethod: input.renderMethod || null,
       enhanceModel: input.enhanceModel || null,
+      customLoadEvent: input.customLoadEvent || false,
+      customLoadEventTriggered: false,
       rawHtml:null
     };
   }
   onMount() {
     this.checkForLazyLoads = this.checkForLazyLoads.bind(this);
     this.doCheckForLazyLoads = this.doCheckForLazyLoads.bind(this);
+    this.doCustomLoadEvent = this.doCustomLoadEvent.bind(this);
     window.addEventListener("scroll", this.checkForLazyLoads);
```

Comment:
if a custom load event exists then we don't want all these other event listeners to even be created.